pkg/satellite/satellitedb/dbx/satellitedb.dbx.go

```diff
@@ -672,7 +672,7 @@ func (Company_CreatedAt_Field) _Column() string { return "created_at" }
 
 type Project struct {
 	Id                []byte
-	UserId            []byte
+	OwnerId           []byte
```

Comment:
`OwnerID` and `ID`. https://github.com/golang/go/wiki/CodeReviewComments#initialisms